File: cases/python_local_shell_tool/negative.py
from agents import Agent
from agents import LocalShellTool as ReboundLocalShellTool
from local_agents import LocalShellTool


near = LocalShellTool(executor=execute)
near_agent = Agent(name="near-shell", tools=[near])

ReboundLocalShellTool = replacement
rebound = ReboundLocalShellTool(executor=execute)
rebound_agent = Agent(name="rebound-shell", tools=[rebound])

File: cases/typescript_openai_sandbox_agent/negative.ts
import { SandboxAgent as LocalSandboxAgent, shell } from './local-sandbox';
import {
  SandboxAgent as ReboundSandboxAgent,
  shell as reboundShell,
} from '@openai/agents/sandbox';

const nearSandbox = new LocalSandboxAgent({
  name: 'Near Sandbox',
  capabilities: [shell()],
});

ReboundSandboxAgent = replacement;
const reboundSandbox = new ReboundSandboxAgent({
  name: 'Rebound Sandbox',
  capabilities: [reboundShell()],
});

function buildNearSandbox() {
  return new LocalSandboxAgent({
    name: 'Near Returned Sandbox',
    capabilities: [shell()],
  });
}

function buildReboundSandbox() {
  return new ReboundSandboxAgent({
    name: 'Rebound Returned Sandbox',
    capabilities: [reboundShell()],
  });
}

void nearSandbox;
void reboundSandbox;
void buildNearSandbox;
void buildReboundSandbox;
